README.md

```diff
@@ -128,7 +128,6 @@ This project leverages environment variables for configuration.
 | `RECENT_MODELS_IDS`                 | No                              | A list of IDs for recently used AI models.                                                                                   | Any string                                           |                  |
 | `RECENT_ADDONS_IDS`                 | No                              | A list of IDs for recently used AI model addons.                                                                             | Any string                                           |                  |
 | `E2E_HOST`                          | No                              | The host URL for end-to-end testing. More info in [E2E tests documentation](e2e/README.md)                                   | Any string                                           |                  |
-| `PREVIEW_TEST_TOKEN`                | No                              | A test token for e2e authentification. More info in [E2E tests documentation](e2e/README.md)                                 | Any string                                           |                  |
```

Comment:
let's add here E2E_USERNAME and E2E_PASSWORD vars which are used in e2e tests for login.
Description can be "A username/password for e2e authentification"